source/sessions/tests/README.rst

```diff
@@ -78,7 +79,21 @@ the given session is *not* marked dirty::
         arguments:
           session: session0
 
+Snapshot session tests
+======================
+Snapshot sessions tests require server of version 5.0 or higher and 
+replica set or a sharded cluster deployment.
+The server ``minSnapshotHistoryWindowInSeconds`` parameter SHOULD be configured to match the test execution time.
```

Comment:
Does this need to be set? Why is this important for the new tests, and by "test execution time", do you mean how long it takes the spec + prose tests to run?

Reply:
We want to ensure that the snapshot the test reads still exists on the server.
IIRC the default history is 10 seconds, which was not sufficient at least for .Net tests. 
So drivers should be aware about this parameter and set it as needed.

Reply:
Why was 10 seconds not long enough? Are there snapshot reads tests that take longer than 10 seconds?

Reply:
It was not sufficient when running in Debug mode.
Apparently it's 5 seconds and .Net tests in release are much quicker. So no changes to EG needed (at least in .Net)

I'd still suggest to track this parameter for such slower dubug mode runs. Not sure what's the best wording though, maybe:
> The server ``minSnapshotHistoryWindowInSeconds`` parameter MAY be configured to match the tests execution time.

?

Reply:
Perhaps something like:
"Note that by default the server only keeps the snapshot alive for 10 seconds. Debugging a driver in the middle of a snapshot session may lead to SnapshotTooOld errors. Drivers can work around this issue by increasing the server's minSnapshotHistoryWindowInSeconds parameter, for example:
```python
client.admin.command('setParameter', 1, minSnapshotHistoryWindowInSeconds=60)
```